plugin/pkg/admission/namespace/lifecycle/admission.go

```diff
@@ -109,24 +109,31 @@ func (l *lifecycle) Admit(a admission.Attributes) error {
 	}
 
 	var (
-		namespaceObj interface{}
-		exists       bool
-		err          error
+		exists bool
+		err    error
 	)
 
-	key := makeNamespaceKey(a.GetNamespace())
-	namespaceObj, exists, err = l.namespaceInformer.GetStore().Get(key)
+	namespace, err := l.namespaceLister.Get(a.GetNamespace())
 	if err != nil {
```

Comment:
nit: I personally prefer a switch, but if we wouldn't touch the pull otherwise I don't care enough to stop it.